src/github/pull-request.ts

```diff
@@ -36,7 +37,7 @@ export const pullRequestWebhookHandler = async (context: WebhookContext, jiraCli
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any
 	let reviews: Octokit.PullsListReviewsResponse = [];
 	try {
-		reviews = await getReviews(gitHubInstallationClient, owner, repoName, pull_request.number);
+		reviews = await getPullRequestReviews(gitHubInstallationClient, context.payload.repository, pull_request, context.log);
```

Comment:
Same here: should we feature-flag this change?

Reply:
Flag added, updated tests to test true/false paths